tests/unit/test_timestream.py

```diff
@@ -657,3 +657,40 @@ def test_batch_load(timestream_database_and_table, path, path2, time_unit, keep_
         """,
     )
     assert df2.shape == (len(df.index), len(df.columns) - 1)
+
+
+@pytest.mark.parametrize(
+    "time_unit",
```

Comment:
Minor - you can avoid having to refer by index `time_unit[0]` by de-structuring the parameter into two variables `time_unit,time_unit_exp` and pass the two as an input to the test case

Reply:
I don't think so, that would create combinations that are not correct (e.g. ("SECONDS", 3) and ("SECONDS", 6) when only ("SECONDS", 1) is correct)

Reply:
There is syntax specifically designed for parameter combinations, here's an example: https://github.com/aws/aws-sdk-pandas/blob/a1957c279afd6e66f1b9fcf3114e4068009cec64/tests/unit/test_athena.py#L323